### DISCUSSION TITLE
Adding/changing email from Android

### DISCUSSION THREAD

**Discussion by bayareaunicorn** [community/community#80862] [Mobile]
Hello I need help adding a email to my account from Android. I lost access to my email address but I am still logged in on the app. I do not have my backup codes but I can still access totp codes for 2fa. I want to add and verify my email for password change or I am open to suggestions on recovery. 🙏

**Comment by AnujaDilrukshiHerath** [comment]
If you've lost access to the email address associated with your email account and want to add and verify a new email address, you can try the following steps.

Using GitHub Website on a Browser:

Open your mobile browser and navigate to the GitHub website.
Log in to your GitHub account if you aren't already logged in.
Go to your account settings.
Adding a New Email Address:

In the account settings, find the section for email addresses.
Add a new email address that you have access to and want to use.
GitHub will send a verification email to the new address.
Verification:

Access your new email account and look for the verification email from GitHub.
Open the email and click on the verification link provided.
Password Change:

After verifying your new email address, you should be able to use it for password recovery.
If you ever need to change your password, initiate the process through the GitHub website. They may send a confirmation email to the verified address.
